packages/website/docusaurusConfig.ts

```diff
@@ -139,6 +139,23 @@ const themeConfig: ThemeCommonConfig & AlgoliaThemeConfig = {
       styles: [],
     },
     additionalLanguages: ['ignore'],
+    magicComments: [
+      {
+        className: 'theme-code-block-highlighted-line',
+        line: 'highlight-next-line',
+        block: { start: 'highlight-start', end: 'highlight-end' },
+      },
+      {
+        className: 'code-block-removed-line',
+        line: 'Remove this line',
+        block: { start: 'Removed lines start', end: 'Removed lines end' },
+      },
+      {
+        className: 'code-block-added-line',
+        line: 'Add this line',
+        block: { start: 'Added lines start', end: 'Added lines end' },
+      },
+    ],
```

Comment:
New Docusaurus feature to add custom metadata to lines—suits this use-case quite well.

I prefer using human-readable identifiers like "Remove this line", but if you want something that stands out more like for machines (e.g. "remove-next-line") just let me know